src/styles/blocks/page.scss

```diff
@@ -0,0 +1,36 @@
+.page {
+  font-family: Space Mono, monospace;
+  color: $main-text-color;
+  font-size: 16px;
+  line-height: 140%;
+  cursor: pointer;
```

Comment:
don't add cursor: pointer for the whole page, you only need to add id to interactive elements (images, buttons etc.)